modelmapper/etl.py

```diff
@@ -248,7 +252,10 @@ def run(self, ping_slack=False, path=None, content=None, content_type=None,
                 self._load(session, data_gen)
         except Exception as e:
             self.report_exception(e)
-            self.logger.exception(str(e))
+            if isinstance(e, CastingError):
```

Comment:
should this just be in its own except block?

Reply:
Good call! will do.